src/commands/app/scaffold.ts

```diff
@@ -0,0 +1,66 @@
+import path from 'path'
+import fs from 'fs'
+
+import { flags } from '@oclif/command'
+import _ from 'lodash'
+
+import { Kommand } from '../../common'
+import { execute } from '../../support/execute'
+
+const templatesDir = path.join(__dirname, '..', '..', 'templates');
+
+export default class AppScaffold extends Kommand {
+  static initSdk = false
+
+  static description = 'Scaffold a new Kuzzle application'
+
+  static flags = {
+    help: flags.help(),
+  }
+
+  static args = [
+    { name: 'name', description: 'Application name', required: true },
+  ]
+
+  async runSafe() {
+    this.logInfo(`Scaffold a new Kuzzle application in ${this.args.name}/`)
+    await execute('rm', '-rf', this.args.name)
```

Comment:
U sure about this? Wouldn't it be better to throw an error and let the user delete the directory?